rwengine/src/ai/PlayerController.hpp

```diff
@@ -55,6 +55,9 @@ class PlayerController : public CharacterController {
     // @todo not implemented yet
     bool isBusted() const;
 
+    void prepareForCutscene();
```

Comment:
Can you add short comments what these function do?

Reply:
Will do